frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -761,15 +784,56 @@ export const CRDCardRow = connect(crdCardRowStateToProps)((props: CRDCardRowProp
   );
 });
 
+const AlertToCreateCustomResource: React.SFC<AlertToCreateCustomResourceProps> = (props) => {
```

Comment:
Hooks cannot be in conditionals and I need to only call the hooks when we actually have a CustomResource in the operator, so I created a component here that only gets called when we have a CR.

Reply:
Let's call this `InitializationResourceAlert`, which matches the annotation and is more clear.

Reply:
We should change `cr` to `initializationResource` throughout.